lib/actions/adapterActions.js

```diff
@@ -1549,11 +1520,11 @@ function closeSelectedAdapter(dispatch, getState) {
  * port COM name and device family.
  *
  * @param {String} portPath Serial port COM name, e.g. COM1, /dev/ttyACM0
- * @param {Number} baudRate The baud rate to use when opening the serial port.
  * @param {Number} sdBleApiVersion SoftDevice version number.
+ * @param {Number} baudRate The baud rate to use when opening the serial port.
  * @returns {function(*)} Function that can be passed to redux dispatch.
  */
-export function openAdapter(portPath, baudRate, sdBleApiVersion) {
+export function openAdapter(portPath, sdBleApiVersion, baudRate = 1000000) {
```

Comment:
You switched the order of the parameters but in these places I think you forgot to switch them:
https://github.com/NordicSemiconductor/pc-nrfconnect-ble/blob/bbda71f789fc0f3b8aab46a0dac3de238adb9a3f/lib/actions/serverSetupActions.js#L429
https://github.com/NordicSemiconductor/pc-nrfconnect-ble/blob/bbda71f789fc0f3b8aab46a0dac3de238adb9a3f/lib/actions/adapterActions.js#L1658

Reply:
Good catch!